src/components/Details/QueryComponent.tsx

```diff
@@ -33,7 +33,6 @@ const Query = ({}) => {
         onFetchQuery(response, query, nodeLabels, dispatch);
       })
       .catch((error) => {
-        console.log(error);
```

Comment:
We actually do want to log when errors happen. But use console.warn (for expected or recoverable errors) or console.error (for major issues).